api/v1/app.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+""" ssss """
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+from os import getenv
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
+
+
+@app.teardown_appcontext
+def teardown(self):
+    """ sss """
```

Comment:
Maybe sss is not a good description for what this function does

Reply:
Done!